vehicle/autoware_raw_vehicle_cmd_converter/CMakeLists.txt

```diff
@@ -30,7 +34,7 @@ if(BUILD_TESTING)
   )
   set(TEST_RAW_VEHICLE_CMD_CONVERTER_EXE test_autoware_raw_vehicle_cmd_converter)
   ament_add_ros_isolated_gtest(${TEST_RAW_VEHICLE_CMD_CONVERTER_EXE} ${TEST_SOURCES})
-  target_link_libraries(${TEST_RAW_VEHICLE_CMD_CONVERTER_EXE} actuation_map_converter autoware_raw_vehicle_cmd_converter_node_component)
+  target_link_libraries(${TEST_RAW_VEHICLE_CMD_CONVERTER_EXE} actuation_map_converter vehicle_adaptor autoware_raw_vehicle_cmd_converter_node_component)
```

Comment:
@kosuke55 why did you merge this without running the required checks?

https://github.com/autowarefoundation/autoware.universe/actions/runs/12449651244/job/34755555845#step:16:18956

Because this line is too long, the test is failing.

We should never merge without passing the required checks.